CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.5.0
+
+**Note**: This release supports [Flutter 3.16.0](https://docs.flutter.dev/release/release-notes/release-notes-3.16.0), for migration details see the [release notes](https://github.com/VeryGoodOpenSource/mockingjay/releases/tag/v0.5.0).
```

Comment:
Referenced the release tag, once tagged I will paste the migration details with the description.